src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisController.java

```diff
@@ -265,7 +266,16 @@ public String getDetailsPage(@PathVariable Long submissionId, Model model, Local
 		// - Paired
 		Set<SequenceFilePair> inputFilePairs = sequencingObjectService.getSequencingObjectsOfTypeForAnalysisSubmission(
 				submission, SequenceFilePair.class);
-		model.addAttribute("paired_end", inputFilePairs);
+		List<SampleFiles> sampleFiles = inputFilePairs.stream().map(SampleFiles::new).sorted((a, b) -> {
+			if (a.sample == null) {
```

Comment:
I think to be complete you'll have to `return 0` initially in the case where both `a.sample` and `b.sample` are null (since they are both equal in that case).

Reply:
Thanks, updated in 9438855b5eecc532a5fad4495cd84bea5f41dbdd